spot-client/src/common/remote-control/BaseRemoteControlService.js

```diff
@@ -135,7 +135,8 @@ export class BaseRemoteControlService extends Emitter {
                     retryOnUnauthorized,
                     roomName: roomInfo.roomName,
                     roomLock: roomInfo.roomLock,
-                    onDisconnect: this._onDisconnect
+                    onDisconnect: this._onDisconnect,
+                    shouldAttemptReconnect: this._shouldXmppAttemptReconnect.bind(this)
```

Comment:
please comment the new options in JS Docs

Reply:
Force pushed to add the js doc.